static/js/constants.js

```diff
@@ -12,6 +12,8 @@ export const ISO_8601_FORMAT = 'YYYY-MM-DD';
 export const DASHBOARD_FORMAT = 'M/D/Y';
 export const DASHBOARD_MONTH_FORMAT = 'MM[/]YYYY';
 
+export const CP1252_REGIX = /^[\u0020-\u00FF]*$/;
```

Comment:
Please name this `CP1252_REGEX` instead of `CP1252_REGIX`. "Regex" is short for "regular expression". I don't know what "regix" means.